packages/google_maps_flutter/google_maps_flutter_ios/pigeons/messages.dart

```diff
@@ -6,8 +6,10 @@ import 'package:pigeon/pigeon.dart';
 
 @ConfigurePigeon(PigeonOptions(
   dartOut: 'lib/src/messages.g.dart',
-  objcHeaderOut: 'ios/Classes/messages.g.h',
-  objcSourceOut: 'ios/Classes/messages.g.m',
+  objcHeaderOut:
+      'ios/google_maps_flutter_ios/Sources/google_maps_flutter_ios/messages.g.h',
```

Comment:
In the packages repo we've been putting the generated Pigeon header in the `include` directory:

```suggestion
      'ios/google_maps_flutter_ios/Sources/google_maps_flutter_ios/include/google_maps_flutter_ios/messages.g.h',
```

Reply:
Done